app/models/upload.rb

```diff
@@ -55,7 +55,7 @@ def self.last_uploads
   end
 
   def self.last_uploads_rows
-    uploads = Upload.last_uploads
+    uploads = Upload.last_uploads.to_a
```

Comment:
What's the warrant for this change being part of the `rails` upgrade?

Reply:
Removed

Reply:
Error from http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fgibct-data-service/detail/rails_5.0.7.2/22/pipeline was reason I originally added it

Reply:
`Upload.last_uploads` returns a `Upload::ActiveRecord_Relation` object in test `spec/controllers/dashboards_controller_spec.rb`

Reply:
I think there was some kind of implicit conversion from `Upload::ActiveRecord_Relation` to array or something because in master branch the test passes fine

I did a `binding.pry()` in `Upload::last_uploads_rows` to check class of uploads object, it shows that `uploads` is a `Upload::ActiveRecord_Relation` and then let me `uploads.push(Upload.new)` without any issue

Reply:
There is a discussion in https://github.com/rails/rails/issues/25906 that talks about how ActiveRecord_Relations are no longer mutable and so the ability to change them was removed in Rails 5